File: kyonggiyo-core/src/main/java/kyonggiyo/domain/auth/exception/ExpiredTokenException.java
package kyonggiyo.domain.auth.exception;

import kyonggiyo.common.exception.AuthenticationException;
import kyonggiyo.common.exception.ErrorCode;

public class ExpiredTokenException extends AuthenticationException {

    public ExpiredTokenException(ErrorCode errorCode) {
        super(errorCode);
    }

    public ExpiredTokenException(ErrorCode errorCode, String loggingMessage) {
        super(errorCode, loggingMessage);
    }

}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/auth/domain/exception/ExpiredTokenException.kt
package kyonggiyo.application.auth.domain.exception

import kyonggiyo.common.exception.AuthenticationException
import kyonggiyo.common.exception.ErrorCode

class ExpiredTokenException : AuthenticationException {
    constructor(errorCode: ErrorCode?) : super(errorCode!!)
    constructor(errorCode: ErrorCode?, loggingMessage: String?) : super(errorCode!!, loggingMessage!!)
}
